sbnanaobj/CMakeLists.txt

```diff
@@ -1 +1,2 @@
 add_subdirectory(StandardRecord)
+add_subdirectory(Morgue)
```

Comment:
Do we want to add some README in the directory explaining what Morgue is for?

Reply:
good shout